openlibrary/templates/search/inside.html

```diff
@@ -22,10 +22,10 @@ <h1>$_("Search Inside")</h1>
             <div class="searchResultsError">$results['error']</div>
 
         $if not num_found:
-            <p class="sansserif red collapse">No hits for: <strong>$q</strong></span></p>
+            <p class="sansserif red collapse">$:_('No hits for: %(query)s', query="<strong>$q</strong></span></p>")
```

Comment:
query is a user provided string; it should be displayed using `$:`, because then someone could craft a URL to run arbitrary JS on a user's computer.

Reply:
Also, `$q` won't get substituted in its current position. You should be able to do something like this:

```
$def escaped_query(): <strong>$q</strong>
$:_('No hits for: %(query)s', query=escaped_query())
```

Reply:
Thanks for the suggestion. I don't think that page was actually tested, so I'll add it to the list to review.